diff --git a/node_modules/@capacitor-community/contacts/android/build.gradle b/node_modules/@capacitor-community/contacts/android/build.gradle
index dc83ed9..477c961 100644
--- a/node_modules/@capacitor-community/contacts/android/build.gradle
+++ b/node_modules/@capacitor-community/contacts/android/build.gradle
@@ -32,7 +32,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
 
